inmem/onboarding_test.go

```diff
@@ -24,5 +24,6 @@ func initOnboardingService(f platformtesting.OnboardingFields, t *testing.T) (pl
 }
 
 func TestGenerate(t *testing.T) {
+	t.Skip("these are no longer being used, using kv inmem instead")
```

Comment:
delete this file

Reply:
https://github.com/influxdata/influxdb/pull/15784#discussion_r343671948